### DISCUSSION TITLE
Problem with GitHub actions

### DISCUSSION THREAD

**Discussion by Aliliche** [cotes2020/jekyll-theme-chirpy#1187] [Q&A]
Hi all, 
Github actions workflow  fails at Test site  with these logs. 
What is the probleme ? 
[6_Test site.txt](https://github.com/cotes2020/jekyll-theme-chirpy/files/12394273/6_Test.site.txt)

the site : https://aliliche.github.io
Thank you.

**Comment by fabse-hack** [comment]
Hey

It was the test site in the deploy yml?
I see already that your site is working, so you solve the problem.

Greetings
Fabi

**Reply by rudzainy** [reply]
@Aliliche I saw that you disabled the failing test. Did that ultimately fixed it for you?

**Reply by abhishek7h** [reply]
@rudzainy I had been facing this issue for months. Commenting the test out fixed it for me!
